commit 7e59066e705035a23aa219d7693f61d200ae378c (refs/changes/68/4868/3)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-09-24T22:23:33+00:00 (1 year ago)
    
    OS-7144 zones stuck shutting down due to zio logging
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/usr/src/cmd/zoneadmd/log.c b/usr/src/cmd/zoneadmd/log.c
index ca531b7177..1afe8e9fae 100644
--- a/usr/src/cmd/zoneadmd/log.c
+++ b/usr/src/cmd/zoneadmd/log.c
@@ -634,6 +634,7 @@ escape_json(const char *sbuf, int slen, char *dbuf, int dlen, int *scntp,
 			if (flushp != NULL) {
 				*flushp = B_TRUE;
 			}
+			break;
 		} else {
 			sbuf++;
 			i += len;
